test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java

```diff
@@ -177,7 +177,7 @@ private AccountID randomNodeFrom(HapiApiSpec spec) {
 			}
 		} catch (Throwable t) {
 			if (!loggingOff) {
-				log.warn(spec.logPrefix() + this + " failed!", t);
+				log.warn(spec.logPrefix() + this.toString() + " failed!", t);
```

Comment:
wouldn't this toString automatically called ?

Reply:
Yeah. It should. I added more log info and forget to clean this up. Fixed now.